### PR TITLE
Transit tube dispensers for JOHN GOBBEL on DISCORD.GG

### PR DESCRIPTION
**Pull request by tralezab**
## About The Pull Request

Creates a subtype of pod stations called pod dispensers. walking into the station will create a pod and place you inside before launching you off.

ATM admin only, could see it as a techweb node

## Why It's Good For The Game

John needs a faster way to get around his map that is spaceproof.

needs sprites, and testing

### CODE REVIEW

**Comment by tralezab** [comment]
got it working, video shortly

**Comment by actioninja** [comment]
liar

**Comment by tralezab** [comment]
i had to edit it and it took me way too long

**Comment by tralezab** [comment]
https://www.youtube.com/watch?v=W62SkxwyDXg&feature=youtu.be

watch out it's really loud, and i mean like hurt your ears loud. you were warned

**Comment by Twaticus** [comment]
thats really awesome. but what happens if it reaches another tube dispenser? it would be sick if the pod gets deleted to prevent a backup of unused pods, which is what transit tubes suffer with currently

**Comment by tralezab** [comment]
in the video, you can see it deletes itself upon reachine it's destination

**Comment by MMMiracles** [comment]
i can't believe it

you actually made transit tubes worth using

**Comment by MrDoomBringer** [comment]
@tralezab You should give these tubes a slightly different sprite to distinguish themselves as temporary tubes, as opposed to the kind that persist after reaching their destination

**Comment by tralezab** [comment]
^^done

File: code/game/objects/structures/transit_tubes/station.dm [hunk 1]
```diff
@@ -209,3 +209,82 @@
...
+	tube_construction = /obj/structure/c_transit_tube/station/dispenser/flipped
+
+/obj/structure/transit_tube/station/dispenser/flipped/init_tube_dirs()
+	..()
+	boarding_dir = dir
+
+
+/obj/structure/transit_tube/station/dispenser/reverse
+	tube_construction = /obj/structure/c_transit_tube/station/dispenser/reverse
+	reverse_launch = 1
```
Comment (ExcessiveUseOfCobblestone) [inline code/game/objects/structures/transit_tubes/station.dm]: t/f?

Comment (tralezab) [reply code/game/objects/structures/transit_tubes/station.dm]: fixed!

**Pull request merged**